config/webpack/webpack.common.js

```diff
@@ -51,7 +51,7 @@ const webpackConfig = {
                 {from: 'assets/css', to: 'css'},
                 {from: 'node_modules/react-pdf/dist/esm/Page/AnnotationLayer.css', to: 'css/AnnotationLayer.css'},
                 {from: 'assets/images/shadow.png', to: 'images/shadow.png'},
-                {from: '.well-known/apple-app-site-association'},
+                {from: '.well-known/apple-app-site-association', to: '.well-known/apple-app-site-association'},
```

Comment:
NAB, probably we can just copy the whole `.well-known` directory

```js
{from: '.well-known'},
```

but what you have is also fine.